ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -219,6 +218,13 @@ public void close() {
   }
 
   /**
+   * Wake up the NetworkClient if it is within a {@link #sendAndPoll(java.util.List)} sleep.
```

Comment:
I think this documentation could be a little more descriptive and perhaps also have a pointer (via the @see tag) to the java nio selector wakeup doc.